docs/using/recipes.md

```diff
@@ -521,29 +521,6 @@ RUN pip install --quiet --no-cache-dir jupyter_contrib_nbextensions && \
 
 Ref: <https://github.com/jupyter/docker-stacks/issues/675>
 
-## Enable auto-sklearn notebooks
```

Comment:
This is easily installable as `mamba install auto-sklearn`, so let's remove this old instruction.